Makefile

```
@@ -1,12 +1,13 @@
 DOCKER_COMPOSE?=docker-compose -f docker-compose.yml
 
-PHONY: build_test_image 
+PHONY: build_test_image
 build_test_image:
 	$(DOCKER_COMPOSE) run --rm postgres -d
 
 PHONY: test
```

checkmake (0.2.2)

[warning] Target "PHONY" should be declared PHONY.

(phonydeclared)